app-mobilephone/heimdall/heimdall-1.4.2-r1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake udev
+
+if [[ ${PV} != 9999 ]]; then
+	SRC_URI="https://github.com/Benjamin-Dobell/Heimdall/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/Heimdall-${PV}"
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Benjamin-Dobell/Heimdall.git"
+fi
+
+DESCRIPTION="Tool suite used to flash firmware onto Samsung devices"
+HOMEPAGE="https://glassechidna.com.au/heimdall/"
```

Comment:
Please also add the Github page as 2ndary HOMEPAGE here.

Reply:
Added for both 1.4.2-r1 and -9999